packages/next-app/src/components/Plot.tsx

```diff
@@ -0,0 +1,47 @@
+import { amberDarkA, blueDarkA, grayDarkA } from "@radix-ui/colors";
+import { merge } from "lodash";
+import PlotRoot, { PlotParams } from "react-plotly.js";
+
+export const defaultAxisLayout = {
+  gridcolor: grayDarkA.grayA10,
+  linecolor: grayDarkA.grayA12,
+  linewidth: 0.5,
+  automargin: true,
+  zerolinecolor: grayDarkA.grayA10,
+};
+
+export const defaultPlotProps = {
+  className: "w-full h-full",
+  useResizeHandler: true,
+  layout: {
+    plot_bgcolor: blueDarkA.blueA1,
+    paper_bgcolor: blueDarkA.blueA1,
+    font: {
+      color: grayDarkA.grayA12,
+      family: "Inter",
+    },
+    xaxis: JSON.parse(JSON.stringify(defaultAxisLayout)),
+    yaxis: JSON.parse(JSON.stringify(defaultAxisLayout)),
+    modebar: {
+      bgcolor: blueDarkA.blueA1,
+      color: grayDarkA.grayA12,
+      activecolor: grayDarkA.grayA12,
+      orientation: "h",
+    },
+    colorway: [blueDarkA.blueA9, amberDarkA.amberA9],
+  },
+  config: {
+    displaylogo: false,
+    showAxisRangeEntryBoxes: false,
+    showSendToCloud: false,
+    showEditInChartStudio: false,
+    showLink: false,
+    watermark: false,
+  },
+  revision: 0,
+};
+
+export default function Plot(props: PlotParams) {
+  const plotProps = merge(JSON.parse(JSON.stringify(defaultPlotProps)), props);
+  return <PlotRoot {...plotProps} key={JSON.stringify(plotProps)} />;
```

Comment:
I'd suggest letting the caller define a key here -- probably not an issue now but plots with the same props would probably lead to a bug in how plotly is handling these plots